uamqp/errors.py

```diff
@@ -259,6 +259,9 @@ def __init__(self):
         response = "Invalid operation: this message is already settled."
         super(MessageAlreadySettled, self).__init__(response)
 
+    def __reduce__(self):
+        return (self.__class__, ())
+
```

Comment:
do we need the override here? seems like it's doing nothing..

Reply:
for some reason when this was removed,  the following results in an error:
```
    internal_message = c_uamqp.create_message()
    internal_message.add_body_data(b"hi")
    message_w_message_param = Message(
        message=internal_message,
        delivery_no=1
    )
```

The error: 
```
E       TypeError: __init__() takes 1 positional argument but 2 were given

```